sample-apps/kafka-app/build.gradle

```diff
@@ -6,12 +6,16 @@ dependencies {
   providedCompile 'javax:javaee-api:8.0.1'
   providedCompile 'org.eclipse.microprofile:microprofile:3.3'
   providedCompile 'org.eclipse.microprofile.reactive.messaging:microprofile-reactive-messaging-api:1.0'
-  compile 'org.apache.kafka:kafka-clients:2.4.0'
+  
+  compile 'org.apache.kafka:kafka-clients:3.6.0'
```

Comment:
This update makes the integrationtest fail. Reverting to 2.4.0 solves that failure. Updating might require altering the test, please examine and solve.

Reply:
The kafka-app integration test passed locally for me, and in the CI build: https://github.com/MicroShed/microshed-testing/actions/runs/6857446341/job/18646588128?pr=319
What failure are you seeing?